src/agent/src/netlink.rs

```diff
@@ -113,12 +114,30 @@ impl Handle {
             self.enable_link(link.index(), false).await?;
         }
 
+        // Get whether the network stack has ipv6 enabled or disabled,
+        // assuming that if the file doesn't exist, it's enabled.
+        let supports_ipv6_all = fs::read_to_string("net.ipv6.conf.all.disable_ipv6")
```

Comment:
It seemed the file path should be "/proc/sys/net/ipv6/conf/all/disable_ipv6" and
"/proc/sys/net/ipv6/conf/default/disable_ipv6"  ?